lib/commons/dom/is-hidden-with-css.js

```diff
@@ -10,6 +10,16 @@
  * @return {Boolean} the element's hidden status
  */
 dom.isHiddenWithCSS = function isHiddenWithCSS(el, descendentVisibilityValue) {
+	const vNode = axe.utils.getNodeFromTree(el);
+
+	if (vNode._isHiddenWithCSS === void 0) {
+		vNode._isHiddenWithCSS = _isHiddenWithCSS(el, descendentVisibilityValue);
```

Comment:
@straker should this not push to `vnode._cache.isHiddenWithCSS`?

Reply:
Only if we were to update all the calls to `isHiddenWithCSS` so they called the vNodes cache function. We did this same thing for [`utils.isVisible`](https://github.com/dequelabs/axe-core/blob/develop/lib/core/utils/is-hidden.js#L23-L25) so we could get all the performance gains without having to update every use of the function.